sdk/include/psdk/d3dkmdt.h

```diff
@@ -24,7 +24,8 @@
 
 #include "d3dukmdt.h"
 
-#define NTSTATUS                int32_t
+#ifndef __REACTOS__
+#define NTSTATUS int32_t
```

Comment:
Maybe add an empty line between the ifndef and the define, for some breathing room (since there's a ton of stuff inside the ifndef block).